### PR TITLE
Fix characters class color in armory

### PR DESCRIPTION
**Pull request by CraftedRO**
before : 
Capturez

after :
Captureafter

### CODE REVIEW

File: application/modules/character/views/character.tpl [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	
 	<section id="armory_name">
 		<h1>{$name} <a href="{$url}guild/{$realmId}/{$guild}">{$guildName}</a></h1>
-		<h2><b>{$level}</b> {$raceName} {$className}, <i>{$realmName}</i></h2>
+		<h2 class="color-c{$class}"><b>{$level}</b> {$raceName} {$className}, <i>{$realmName}</i></h2>
```
Comment (OMGhixD-OG) [inline application/modules/character/views/character.tpl:18]: Would suggest

`<h2 class="color-c{$class}"><b>{$level}</b> {$raceName} {$className}</h2> <i>{$realmName}</i>`

instead as we dont want the class color to apply to the realmname as well.

Comment (OMGhixD-OG) [reply application/modules/character/views/character.tpl:18]: And the RealmName are usually something that **can** appear on a new line and by itself.

Comment (CraftedRO) [reply application/modules/character/views/character.tpl:18]: how you wish , idk you can edit how you think , but the realm on the next line is kinda ugly

**Review by OMGhixD-OG — COMMENTED** [review]
Thank you for your contribution

**Comment by OMGhixD-OG** [comment]
Thank you for your contribution

**Pull request merged**